packages/api/src/repository/feed.ts

```diff
@@ -0,0 +1,33 @@
+import { ILike } from 'typeorm'
+import { appDataSource } from '../data_source'
+import { Feed } from '../entity/feed'
+
+export const feedRepository = appDataSource.getRepository(Feed).extend({
```

Comment:
For this do we need to use user id to fetch their private feeds, or add a filter for public/private?

Reply:
This table is actually used for searching a feed and we have `is_public` flag in the `subscription` table

Reply:
I see, so when a user adds an RSS subscription is a feed row created?

Reply:
Yeah and we need to migrate the existing data too